misc/testerina/modules/testerina-core/src/main/ballerina/src/test/annotations.bal

```diff
@@ -62,7 +62,7 @@ public annotation BeforeEach on function;
 public annotation AfterEach on function;
 
 # Identifies the MockFunction object
-public const annotation MockConfig MockFn on source var;
+public const annotation MockConfig Mock on source var;
 
 # Identifies the Mock Function
-public annotation MockConf Mock on function;
+public annotation MockConf MockFn on function;
```

Comment:
This change will break the BBE tests in ballerina-distribution repo. Can we have the same Mock annotation for both function and parameter.

Reply:
Changed the annotation to `Mock` for both old and new implementations